core/serialization/download_xlsx_serializer.py

```diff
@@ -0,0 +1,660 @@
+                    organisation_metadata[model.programme.programme_id],
+                ).to_dict()
+            )
+        else:
+            # project risks
+            output.append(
+                RiskRegisterSerializer(
+                    model,
+                    places_metadata[model.project.project_id],
+                    organisation_metadata[model.project.project_id],
+                ).to_dict()
+            )
+
+    return output
+
+
+class OrganisationSerializer:
+    def __init__(self, organisation: ents.Organisation):
+        self.organisation = organisation
+
+    def to_dict(self):
+        return {
+            "OrganisationID": str(self.organisation.id),
+            "OrganisationName": self.organisation.organisation_name,
+            "Geography": self.organisation.geography,
+        }
+
+
+class ProgrammeSerializer:
+    def __init__(self, programme: ents.Programme, place: str, organisation_data: str):
+        self.programme = programme
+        self.place = place
+        self.organisation_data = organisation_data
+
+    def to_dict(self):
+        return {
+            "ProgrammeID": self.programme.programme_id,
+            "ProgrammeName": self.programme.programme_name,
+            "OrganisationName": self.organisation_data,
+            "Place": self.place,
+            "FundTypeID": self.programme.fund_type_id,
+        }
+
+
+class ProgrammeProgressSerializer:
+    def __init__(self, programme_progress: ents.ProgrammeProgress, place: str, organisation_data: str):
+        self.programme_progress = programme_progress
+        self.place = place
+        self.organisation_data = organisation_data
+
+    def to_dict(self):
+        return {
+            "SubmissionID": self.programme_progress.submission.submission_id,
+            "ProgrammeID": self.programme_progress.programme.programme_id,
+            "OrganisationName": self.organisation_data,
+            "Place": self.place,
+            "Question": self.programme_progress.question,
+            "Answer": self.programme_progress.answer,
+        }
+
+
+class PlaceDetailSerializer:
+    def __init__(self, place_detail: ents.PlaceDetail, place: str, org: str):
+        self.place_detail = place_detail
+        self.place = place
+        self.org = org
+
+    def to_dict(self):
+        return {
+            "SubmissionID": self.place_detail.submission.submission_id,
+            "ProgrammeID": self.place_detail.programme.programme_id,
+            "OrganisationName": self.org,
+            "Place": self.place,
+            "Question": self.place_detail.question,
+            "Indicator": self.place_detail.indicator,
+            "Answer": self.place_detail.answer,
+        }
+
+
+class FundingQuestionSerializer:
+    def __init__(self, funding_question: ents.FundingQuestion, place: str, org: str):
+        self.funding_question = funding_question
+        self.place = place
+        self.org = org
+
+    # TODO ProjectName
```

Comment:
should this be left as a TODO? In ticket https://trello.com/c/uWubpFxg it says this has been done (item crossed off)

Reply:
The ticket says this:
> FundingQuestions: Remove Guidance Notes, added ProjectID. Note:  just fyi, some of these are programme level Qs and some are Project level. So Project Name may be blank

We can't add ProjectID to this as it's programme level data... 

Ideas?

Reply:
Assuming this is just wrong on the ticket.. I've removed the TODO